### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by Prabhatm09**
sir please review it and merge it

### CODE REVIEW

File: src/components/login/ResetPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        const data = new FormData(e.currentTarget);
+        const actualData = {
+            oldpassword: data.get("Oldpassword"),
+            password: data.get("newpassword"),
+            password_confirmation: data.get("password_confirmation")
+
+        };
+
+        if (actualData.oldpassword && actualData.password && actualData.password_confirmation) {
+            console.log(actualData);
```
Comment (Souvikdeb2612) [inline src/components/login/ResetPassword.jsx]: remove/comment console statement

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import InputBase from "@mui/material/InputBase";
+import Badge from "@mui/material/Badge";
+import MenuItem from "@mui/material/MenuItem";
+import Menu from "@mui/material/Menu";
+import MenuIcon from "@mui/icons-material/Menu";
+import SearchIcon from "@mui/icons-material/Search";
```
Comment (Souvikdeb2612) [inline src/components/navbar/Navbar.jsx:13]: Format imports

File: src/components/navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+}));
+
+export const Navbar = ({ handle, setMode, mode }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [mobileMoreAnchorEl, setMobileMoreAnchorEl] = React.useState(null);
+
+  const isMenuOpen = Boolean(anchorEl);
+  const isMobileMenuOpen = Boolean(mobileMoreAnchorEl);
+
+  const handleProfileMenuOpen = (event) => {
```
Comment (Souvikdeb2612) [inline src/components/navbar/Navbar.jsx:122]: can be written as one single line

File: src/components/navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleMobileMenuClose = () => {
```
Comment (Souvikdeb2612) [inline src/components/navbar/Navbar.jsx]: same as above

File: src/components/sidebar/Sidebar.jsx [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+import MuiDrawer from "@mui/material/Drawer";
+import List from "@mui/material/List";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import Collapse from "@mui/material/Collapse";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
```
Comment (Souvikdeb2612) [inline src/components/sidebar/Sidebar.jsx:13]: format imports

File: src/components/sidebar/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+  ...(!open && {
+    ...closedMixin(theme),
+    "& .MuiDrawer-paper": closedMixin(theme),
+  }),
+}));
+
+export const Sidebar = ({ open, handleDrawerClose }) => {
+  const [down, setDown] = React.useState(true);
+
+  const handleClick = () => {
```
Comment (Souvikdeb2612) [inline src/components/sidebar/Sidebar.jsx]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/navbar/Navbar.jsx (hunk 3 above)
Comment (Prabhatm09) [reply src/components/navbar/Navbar.jsx:122]: sir this MUI code not i write

File: src/components/sidebar/Sidebar.jsx (hunk 6 above)
Comment (Prabhatm09) [reply src/components/sidebar/Sidebar.jsx]: done

**Pull request merged**